### ISSUE TITLE
Extension not working for the latest JupyterLab

### ISSUE THREAD

**Issue by gaow** [vatlab/jupyterlab-sos#22]
Just got this error message:

```
$ jupyter labextension install jupyterlab-sos
jupyterlab-sos-0.2.2.tgz


   ╭─────────────────────────────────────╮
   │                                     │
   │   Update available 5.6.0 → 6.4.0    │
   │     Run npm i -g npm to update      │
   │                                     │
   ╰─────────────────────────────────────╯


Errored, use --debug for full output:
ValueError: This extension does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension        Package
>=0.18.2 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/application
>=0.18.2 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/apputils
>=0.18.2 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/notebook
```

My JupyterLab is at 0.34.0

**Comment by BoPeng** [comment]
Yeah, they just released 0.34.0 yesterday and I have to bump `jupyterlab-sos` version to match them. I will just blindly update the versions because I do not have time to verify if `jupyterlab-sos` is compatible with `JupyterLab 0.34`.

**Comment by BoPeng** [comment]
Try again.

**Comment by gaow** [comment]
Seems to work!

**Comment by gaow** [comment]
Needs another version bump here?

```
Errored, use --debug for full output:
ValueError:
"jupyterlab-sos@0.2.6" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLab              Extension        Package
>=0.19.1 <0.20.0        >=0.18.2 <0.19.0 @jupyterlab/application
>=0.19.1 <0.20.0        >=0.18.2 <0.19.0 @jupyterlab/apputils
>=0.19.2 <0.20.0        >=0.18.2 <0.19.0 @jupyterlab/notebook
```

**Comment by BoPeng** [comment]
Done. Another "blind" upgrade. Let me know if there is any  compatibility issue.

**Comment by gaow** [comment]
Did you push?

**Comment by BoPeng** [comment]
Did you use `npm` to get the latest?

**Comment by gaow** [comment]
I see. Tthe installation actually worked from npm. But not with `./build.sh`

**Comment by BoPeng** [comment]
Oh, I pushed later.

**Comment by gaow** [comment]
A similar problem:

```
> /home/gaow/Public/miniconda3/bin/npm pack /home/gaow/GIT/software/jupyterlab-sos
jupyterlab-sos-0.4.4.tgz

Errored, use --debug for full output:
ValueError: 
"jupyterlab-sos@0.4.4" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLab              Extension        Package
>=0.19.1 <0.20.0        >=1.0.0-alpha.3 <2.0.0@jupyterlab/application
>=0.19.1 <0.20.0        >=1.0.0-alpha.3 <2.0.0@jupyterlab/apputils
>=0.19.1 <0.20.0        >=1.0.0-alpha.3 <2.0.0@jupyterlab/console
>=0.19.2 <0.20.0        >=1.0.0-alpha.3 <2.0.0@jupyterlab/notebook
```

I'm on `master` of SoS and SoS notebook and JupyterLab-SoS. I used `./build.sh` and got this error. However installation runs fine with `npm` update. But it does not work after I updated and open JupyterLab. It just keep crashing saying jupyterlab needs to rebuild and remove jupyter-sos.

**Comment by BoPeng** [comment]
I had to use the alpha version of JupyterLab to develop the rest of the features of jupyterlab-sos. Please upgrade your jupyter lab with commnad

```
pip install jupyterlab==1.0.0a1
```
if you have enough confidence on an alpha version.

**Comment by gaow** [comment]
Sure -- alpha is good. Just updated. All working fine now! (except my toc extension is gone, possibly need reinstall)

**Comment by BoPeng** [comment]
Too bad most extensions are not available for 1.0.0 alpha...
